cmd/hemictl/hemictl.go

```diff
@@ -261,18 +261,17 @@ func tbcdb() error {
 
 	case "blocksmissing":
 		count := args["count"]
-		c, err := strconv.ParseUint(count, 10, 64)
+		c, err := strconv.ParseInt(count, 10, 64)
```

Comment:
This is potentially cast to a 32-bit integer (depending on the platform) below, instead of parsing as a 64-bit int and casting to an int - could we parse as a 32-bit integer? I'm not sure we will need to use count > `4,294,967,296`.
```suggestion
		c, err := strconv.ParseInt(count, 10, 32)
```